partition/bootloader.go

```diff
@@ -43,9 +36,11 @@ const (
 	// Initial and final values
 	bootloaderBootmodeTry     = "try"
 	bootloaderBootmodeSuccess = "regular"
+)
 
-	// textual description in hardware.yaml for AB systems
-	bootloaderSystemAB = "system-AB"
+var (
+	// ErrBootloader is returned if the bootloader can not be determined
+	ErrBootloader = errors.New("Unable to determine bootloader")
```

Comment:
s/Unable to/cannot/

The review is mainly due to the lowercasing, but for some reason these sentences always sound better with "cannot".. perhaps it's just me, though.